PendingReleaseNotes.md

```diff
@@ -9,6 +9,7 @@ an example usage
 - Added K8s 1.15 to the test matrix and removed K8s 1.10 from the test matrix.
 - OwnerReferences are created with the fully qualified `apiVersion` such that the references will work properly on OpenShift.
 - Linear disk device can now be used for Ceph OSDs.
+- The integration tests can be triggered for specific storage providers rather than always running all tests. See the [dev guide](INSTALL.md#test-storage-provider) for more details.
```

Comment:
Use full link instead? `https://github.com/rook/rook/blob/master/`

Reply:
This relative link will be fine for now, we'll just need to check the links when we create the release notes, which will use the 1.1 branch.